http/src/main/scala/org/http4s/blaze/http/http2/server/RequestParser.scala

```diff
@@ -1,12 +1,14 @@
 package org.http4s.blaze.http.http2.server
 
-import org.http4s.blaze.http.http2.StageTools._
 import org.http4s.blaze.http.{BodyReader, HttpRequest, _}
 
 import scala.collection.mutable.ArrayBuffer
 
 /** Tool for turning a HEADERS frame into a request */
 private object RequestParser {
+  import org.http4s.blaze.http.http2.PseudoHeaders._
+  import org.http4s.blaze.http.http2.StageTools._
```

Comment:
Any reason not to put these with the rest of the imports?

Reply:
Personal preference I suppose. I'll move it.